src/components/NewsletterForm/index.tsx

```diff
@@ -46,7 +46,7 @@ export const NewsletterForm = ({ className = '', placement }: NewsletterFormProp
         >
             <div className="text-center hidden md:block">
                 <CloudinaryImage
-                    src="https://res.cloudinary.com/dmukukwp6/image/upload/posthog.com/src/images/swole-hog.png"
+                    src="https://res.cloudinary.com/dmukukwp6/image/upload/engineer_47d6638eae.png"
                     objectFit="contain"
```

Comment:
Swole hog is more eye catching IMO. Music hog doesnt jump out really. Would keep swole hog

Reply:
He is more eye-catching, but I do find swole hog's t-shirt hard to read; engineer hog is more visually polished but yes, muted.

Astronaut hog, call to adventure?

<img width="211" alt="Screenshot 2024-11-13 at 9 58 52 AM" src="https://github.com/user-attachments/assets/43b58493-53d8-42c4-a4ee-cbecb8109ccb">